codebase2/codebase-sqlite/U/Codebase/Sqlite/Queries.hs

```diff
@@ -880,6 +882,24 @@ removeHashObjectsByHashingVersion hashVersion =
       WHERE hash_version = ?
 |]
 
+addHashObjectConstraintToCausalTable :: DB m => m ()
+addHashObjectConstraintToCausalTable =
+  execute_ sql
+  where
+    sql =
+      [here|
+-- Not sure why, but we need this savepoint or this operation fails.
```

Comment:
I need to explore this a bit further, without this savepoint we get a "TABLE causal does not exist" error later on...